docs/features/deserializing-events.md

```diff
@@ -24,6 +25,8 @@ When using official Azure events, you can use `.ParseFromData<>` to deserialize
 EventGridBatch<EventGridEvent<StorageBlobCreatedEventData>> eventGridBatch = EventGridParser.ParseFromData<StorageBlobCreatedEventData>(rawEvent);
```

Comment:
The "Or from event data type objects where the event data payload is available via the `.GetPayload()` method." section above fields weird.

Is this intentional?

Reply:
Yeah, I see it. Not sure exactly what it does there. Maybe something during merging or so? I we can remove that sentence?

Reply:
👌, is now removed!